plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -398,24 +396,6 @@ function BookInfoManager:getBookInfo(filepath, get_cover)
 end
 
 function BookInfoManager:extractBookInfo(filepath, cover_specs)
-    -- This will be run in a subprocess
-    -- We use a temporary directory for cre cache (that will not affect parent process),
-    -- so we don't fill the main cache with books we're not actually reading
-    if not self.cre_cache_overriden then
-        -- We need to init engine (if no crengine book has yet been opened),
-        -- so it does not reset our temporary cache dir when we first open
-        -- a crengine book for extraction.
-        local cre = require("document/credocument"):engineInit()
```

Comment:
Just asking: sure this `:engineInit()` is not needed (for maybe other things than the cache)? (when you start with filebrowser so crengine is not initialized, and you go at extract metadata from an EPUB).

Reply:
It's called automatically when loading a document.

Reply:
When fully loading ? or when loading with metadataonly ?
My question is whether, if it is not loaded when metadataonly, and a full loading has not yet been done, could it be needed ?

Reply:
`Document:new` calls `CreDocument:init` which calls `CreDocument:engineInit`.